### ISSUE TITLE
Snap to Edges - dragging to top does not fill entire screen - after update to 0.16.0

### ISSUE THREAD

**Issue by boksir** [nothingislost/obsidian-hover-editor#170]
When I drag a Hover Editor PopOver to the top of the screen, it doesn't fill the entire screen. About 20% of the screen is not covered. Also found that dragging the sides of the popover horizontally, doesn't allow the horizontal drag to more than 80% of the horizontal screen span.
This is irrespective of the theme.. same problem in default theme as well

I am on MacOS

**Comment by pjeby** [comment]
This is due to a CSS change in Obsidian 0.16, and it will be fixed in the next Hover Editor release (0.11.6 or later), which is currently a work in progress.

In the meantime you can use the following CSS snipped to work around it:

```css
.popover.hover-editor.snap-to-viewport {
  max-width: unset !important;
}
```

(The issue is that Obsidian 0.16 added a `max-width: 80vw` rule that prevented any popovers from being wider than 80% of the window.)

**Comment by boksir** [comment]
Thank you PJEby!!
